src/app/blocks/mapping-block/mapping-util.spec.ts

```diff
@@ -86,4 +86,12 @@ describe('MappingUtil', () => {
 
     expect(mappingUtility(data, expr)).toBe(expected);
   });
+
+  it('should parse a valid JSON string into an object', () => {
+    const data = { jsonString: '{"name": "Alice", "age": 30}' };
+    const expr = "jsonParse(jsonString)";
+    const expected = { name: "Alice", age: 30 };
+
+    expect(mappingUtility(data, expr)).toBe(expected);
+  });
```

Comment:
This test throws an error unfortunately - 

`Expected Object({ name: 'Alice', age: 30 }) to be Object({ name: 'Alice', age: 30 }). Tip: To check for deep equality, use .toEqual() instead of .toBe()`

Fortunately that's very clear! I checked, and the change resolves the error.